yt/yt/tests/integration/node/test_job_proxy.py

```diff
@@ -447,3 +450,103 @@ def test_separate_directory(self):
                 "job_proxy.log"
             )
             assert os.path.exists(log_path)
+
+
+class TestDumpJobProxyLog(YTEnvSetup):
+    NUM_MASTERS = 1
+    NUM_NODES = 2
+    NUM_SCHEDULERS = 1
+    NUM_CONTROLLER_AGENTS = 1
+    USE_DYNAMIC_TABLES = True
+
+    JOB_PROXY_LOGGING = {"mode": "per_job_directory"}
+
+    DELTA_NODE_CONFIG = {
+        "exec_node": {
+            "job_controller": {
+                "resource_limits": {
+                    "user_slots": 5,
+                    "cpu": 5,
+                    "memory": 5 * 1024 ** 3,
+                }
+            },
+        },
+        "job_resource_manager": {
+            "resource_limits": {
+                "user_slots": 5,
+                "cpu": 5,
+                "memory": 5 * 1024 ** 3,
+            }
+        },
+    }
+
+    DELTA_SCHEDULER_CONFIG = {
+        "scheduler": {
+            "watchers_update_period": 100,
+            "operations_update_period": 10,
+            "operations_cleaner": {
+                "enable": False,
+                "analysis_period": 100,
+                # Cleanup all operations
+                "hard_retained_operation_count": 0,
+                "clean_delay": 0,
+            },
+            "enable_job_reporter": True,
+            "enable_job_spec_reporter": True,
+            "enable_job_stderr_reporter": True,
+        },
+    }
+
+    DELTA_DYNAMIC_NODE_CONFIG = {
+        "%true": {
+            "exec_node": {
+                "job_reporter": {
+                    "reporting_period": 10,
+                    "min_repeat_delay": 10,
+                    "max_repeat_delay": 10,
+                },
+            },
+        },
+    }
+
+    def setup_method(self, method):
+        super(TestDumpJobProxyLog, self).setup_method(method)
+        sync_create_cells(1)
+        init_operations_archive.create_tables_latest_version(
+            self.Env.create_native_client(), override_tablet_cell_bundle="default"
+        )
+
+    def teardown_method(self, method):
+        remove("//sys/operations_archive", force=True)
+        super(TestDumpJobProxyLog, self).teardown_method(method)
+
+    @authors("tagirhamitov")
+    def test_rpc_method(self):
```

Comment:
It is better to name `test_dump_for_running_job` and test below `test_dump_for_finished_job`